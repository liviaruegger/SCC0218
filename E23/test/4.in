100
bzzaybzzbyaybybzzbbzzyyzayaaabbzbzbabbzzbaayyzbbay
azbzabbyyabybbaybzzzzbazzazabyyyazybzzbzyaabbbayyb
bayzzzzyzabzaabazzayyaabayyyyzybyyabzbbbbzayazaza
aabzyazyayybyzbazazayyayazabazazyzabybzzabaayayzbz
baybyyybybbzbaabzbyayzyyayzyyazbyabbaaabzbaaaayzb
zazbbbzbzabzzayyzbybyzbyabbzzbyabzybzbzbabbazbby
zbzyzabzabyzazybbbzbzazbzbaabbyyybybbzzzbzzaaaaa
bzbzzyyabbybyayzzbzyayzbazabayzzybaybabzyyzaazzzya
yyabyaabyyybaybyzaaybaayabybbyybbzabbyayybybzzbb
yzaazybabbzabzzazbybzbbzzaabbzybzyzzaybbyzyzabybaz
ayabaayzybzyyzzaybzybzazayyzayazyayazbbybbzazbya
azbabbzzazyyayzayyyzzaaazbaazzazyzaybybzzabyabbbz
azyzbbaazabyazzaayayyabbyyyayzyzyyyyzzbyyaybzazyzz
ybbazzyazzyyabzbybybzbyzaabbaabbbbazyayzazzzzzzaaz
abyzbbzbbyabbyayzyybbazabzayzayybzbzaazbyaybyaaa
azyabyzzayyyzbzzyaabybzybbzbzyazayaayazazbyzyybza
yzbyyybbyybyzbbzyazbyyabzyyzbaabzyzbybaaybaybyzb
yazybayazabazzbyzbaazbbbbazbzyzbazbbzybaaaazzzbzz
baaybbzzzazaazyybbbyaabbbazayaazyazazbybayzbybzay
zyabbyzyayabbbyabyzaybyaabbbyyaayybazzazbyyaababza
bzzzbyyyzzzyyayaabazbyybbyyyyazaybabzabzbaaayzaaz
zyzyaabyaybabzzzzyzbyabaayaaabyabaabbbayybyazbzbyb
zzzybyzabyybayyzbyzzazyaabazabbbaaazyyzzybayzaza
yybyybazayaybzybayybzybbyyzybzzyzyybayaazazyybyba
zaabayzzbbyzyaazyyaaaybyayzyyaaazzaabbzzbyabzyby
bayaayazaybbyzzabbyybazbbbbabzzybyyababayzyzbyyzy
aabyyyayybbababbyaabbbbzyzbbbbyabyazbbbbaazyzzazaz
yybzzaaayabbzzzybzbaayzyyzbzbaabyyayaabayzzybaaz
bzyzzzybbbbyyybzabbyybyabazbzzbzybazybabzbyyaaayyb
azbbbyayyaaaybyzbaabbbzbayybazabbbaaabyazbyzzybza
ayabzaazbyybzzababaayyyyyabzabazayzazaayzzabzzzbbz
baaayayazaazbyzbzzyyzzbbbzyzbaazazzyyaaayzbbyaazaa
yzzzyyabaazzzbzzaayazzaababzbybzyyzaazzyyybbaabz
azybayayzyyzabyyzzyyyaabayaayyaybybbbaybbabayyza
abybabazazbayybzbyyabybzabzzzaybzazzbabazyzzbyzbab
zazaaazyzaayzyzybzbbyzzbzazzzzazbazzzbazyyyabyazy
zyzbzabzybazzyzbyzzzyzyyzazzybbayzzayabyyzabzbaay
bzzzyyaazzyzaaybyyybbzaybayayzabbabaazabyzzazzbzy
bzzzzzzybzbabzabybaazbzabbzaabbayaybayzzbbzaaaybb
zbzzzbaybybybyayayazyabbaaybbzzabzbyyzazabaazbbzaa
bybybzbazzzabybybyyyzzybyyzayyayyazazbbzbyayzayaab
yyabyaazyyzyazbbzbbyaazaabbyaaayzayzayabyabybybab
zzzzyybbbbzazybabababazzyybyzyzyyaayaybzyzbaybyzb
ybbzbyzzbybzzbzzbabbabyzbaaabbzzabazbaaybaaazzzbbz
bbyyazabyzbbzybbyabzybzzbbzzyaabbbzbbzbbzbbzzyyaby
azbybyzzbazaazbbazybzyzyayyyzbaazayyabyyazayayzb
bzbabzzbbybayzzayyzbbbabyzbzbabybybbabzazbbzzyzbza
byyybbzbabybazzzayabbaazbyyyybybzyzbbayybbzbayzyz
zzabbybbyzbayabbbybaabbzzzbzzbbzbzbayzzzbbyyyzbab
zbaabzababyayzbbabzaybzzaayzaazbyybaybzyyzbbzbzy
bzzbazbzbzaabyybbyzzyybayaazaaabybazbbyzabzazyayby
babbbayyaabbbbbazayyyabbyyabzyybazyaaayzybbbyyyy
zazzzyaayyabzyazzyyzbzybzbaybbzyyyzazbbzabbazabb
ybbyaybyazzzzazzbazzzabazayazyyzyyazyzbaybzabbbaz
azyzayyyaaayyazbyyabyyayayaybbbbzyyyyyybyyzazaba
baazzayazzybbzyazyyyzyaybbyayazyzbayabayazbbzzaz
zzazzbbybyzzaayyzyayzbayzzyzbbzzybaayazayzbazzabaz
zyabbzzayazabzbzyabbbabbbzbbzayzybzyyzbabbyyyybby
bzbyyazabzzzaaaazabzyzbyazybzbzaybzabyabbaabaayb
yayyzbababybayazzbazzybzzaabyaazzyzzyzyyaayazzzbz
bzyabbbyzazzzbbaazzbayzazybazzzbabbbaazybabbyyzaz
bayazyzyzabazzyzazaazyabyzyabayzzzayayyzyzzayzybzz
bbzabazyaazbyzyaazyyazybazbybyyybybabayyzzzayzyyy
yybybybazybazyzzzbyzbayzbyzbbaabzbzbazyzzazbybbaay
byazzyzzazabzbzazazabbayybyyazbazzbaayzybazaazzyzz
ybyyaaayyabzzbyyyyzyzbyaybyzbazybazayzybyyayzyaab
azzaybyaayyaybbyazbyybyzybzyabyzayzazabyzazbzzzby
abbbabayyyyzbzbyaaaabazyazzzbzaabyyybaazyybyyzayz
babzybyzbabybazaaazyaybzyzbbzyyzzbzabzyybyyyyzab
baaazybzybzazyabyyzzzbaazzazabyzzbzaazzaabzzybzyz
bzybyyzzzyzyybbyyybyaayzbazbyazzzbzbaabazyzybazab
aaazbzzyybbabybyyybzbayzabzzazyybaabzzzbbyaayyaz
byazaababzbazbayzayzybbbabzbyazbayyybazazzbbyaya
abyyyabzabbyybyazbzbyyayaayybyaabazyzzbyazbyyzzbaz
yaybbbaababyzzyzbbyzzzabbzaayyyzbaazbazbbbybbbzba
bbaabayzbazzzazzbyayybzbabbzzazyababzazaabybbazabz
byyyzyzbzyyayayabyyyyazybaazazyayzababzbyyzbaaza
yabzayyayyyzaabyzabzazbyyababzbzbyayzazzzzabybya
abbyyabzbyaybaayaazayzyaabzzzayyazzbyaabyabybabb
bbzabbzbaabaabyzaybyabyzabbzbzyzbazbzaaybzbybyabb
bzyzyzzayzzzababzbyzzabbaabzbaabaazbayzabybazyza
yybyyaybaazyzbbzzayayyyabyaazyazzaayzbyabbzzazbyyy
yybbaabayyyyzyyaabzyzybbyybzaaaaaabazabbazzzbaybzb
zaabbzzbzzbzyyybabyyzabybyyazbbzyzbybaybaayzbyabbb
aybzybbyyayzzbyyyaaayzbaybybbzyybybzzzayyzybaaayz
zazyzybyybbzbaybyyybazbbzbzyazbbaaybzazyabbayayaz
bayyzyyzbbbaabbzbayabzyaabaazazzaybbbzzbzaazyzzzy
yazybyayyyybzyabyaaaazyabyzyyyybzabazabzzabbyabzb
zbabazzbazyzybzazyabyzzaybyzaazyyzzbzababbbzbyyyz
zyzzbbbyaayzazzyaybyzzyzyyabbazzyzyzaayyybabybbb
zyzaabyzzbaybybybbazyabbzbaabaaaabbzybazyyazybaaby
aybbzaabyyybbbyzbabbayyyzbbzyyaybyzaaazyazbbybyz
ayazazbbzzaazbaaaazbzayabayzyazaaabzaazbbayybybya
aazbzayyzzzayazzybbababazyzabazyybzybyzyababzazyza
ayyybzaaayzzzazzazzaybbayyyazzbbbyzzabyzbazaabab
zbyaabaaybaabbzbyabbzybaaybbyyaaybzbzyabyyazbayzyz
ybzyayzabayzzzbzzzzyzazayababaybbazbyzzzzbzazabybb
zbybazyzybaababbaaybaaayyayybazyyzyyayzbyazayzyya
ybzyzzzzabaaayyzzbzazabyyazybzbbyaazbbbzybyzyazzyb
yyzybazyaabbyyaayayyzbabaayzayyzbbbyayaaybzzaaba
