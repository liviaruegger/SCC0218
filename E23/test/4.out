ayazazayazbzazayazayazaybzbzbzayayazazayazbzayazbz
byaybzazazazazbzayayazbyazazazazbyazayayazbzazbzaz
azayayazazaybzazaybzazbzbzazayazazbzazazaybzbybyb
bzayazazbzazayazazazazbzbybzbybyaybzazaybzbzazayay
azazazazazayazbzazazayazbzazazazazazbzbzazbzbzaya
azazazazazayazazazazayazbzayazazayazazazbzazazaz
azazazaybzaybyazazazazazazbzazazazazayayayazbzbz
ayayazazazazazayazazbzazbybzbzayazbzazayazazbyayaz
azbzazbzazazbzazazbzazbzbzazazazaybzazbzazazayaz
aybzazazazazayazazazazayazbzayazazaybzazayaybzazby
bzbzbzayazazayazazazaybybzaybzbyazazazazazazazaz
byazazaybyazbzazazayazbzazbzaybyaybzazayazazbzaza
byayazbzazazbyazbzbzazazazazayayazazayazazazazazay
azazayazayazbzazazazazaybzazbzazazbyazaybyayayazby
bzayazazazbzazbzazazazazaybzazazayaybzazazazazbz
byazazaybzazazayazbzazazazazazbybzbzazazazayazayb
ayazazazazazazayazazazbzazayazbzazazazbzazbzazaz
azazazazazazayazazbzazazazazazazbyazazazbzbyayaya
azbzazayazazbyazazazbzazazazazbyazazazazbzazazaza
azbzazazbzbzazazazazazazbzazazbzazazaybyazazbzbzaz
ayayazazayazazazbzbyazazazazazazazbzazayazbzaybza
azazbzazbzazayayazazazazbzbzbzazazbzazbzazazazazaz
ayazazazazazbzayazaybyazbzbybzazbzbyazayazbzazaz
azazazbybzbzayazbzazazazazazayazazazbzbzazazazazb
azbzbzayazayazbyazbzbzazbzazazbzaybzazayazbzazaz
azazbzbybzazayazazazazazazazayazazazazazayayazaya
bzazazbzazazazazazbzazayayazazazazbyazazbzazaybyby
azayazbzazazayazayazbzazayayazbzazbzbzazayazazby
ayayayazazazazaybzazazazazazayayazbyazbzazazbzbzaz
byazazbzazbzazayazbzazazbzazbybzazbzbzazazayazayb
bzbzazbyazazaybzbzbzazazazaybzbybzazazbzaybzayazay
azbzazazazbyazazayazayazayayazbybyazazbzayazazbybz
ayayazbzbzayazaybzazaybzazayazayazazbyazazazbzay
byazbzbzazaybzazayazazbzbzbzazbzazazazazazazazaz
bzazbzbybyazazayazazazaybzayazazazayazazazayazazbz
azazbzazazbzazazayazayazazayaybyazayazbyazazazbya
azazazayazbyazazayayayazazayazazayazazazaybzazbza
ayayazbzayaybzazazazaybzazazaybzazazbybzayazayaya
ayayayazayazaybzazbzazazazazbzazazazbzayazazbzaza
azayazbzazazazbzbzbyazazbzazayazayazaybybzbzazaybz
azazayazayazazazazazayazazazazbzazazazayazbzazazbz
azbzazbyazazbyazazazbzazbzazbzbzazaybzbzazazazaza
ayayazazazazazazazazazayazazazazazbzbzayayazazaya
azayazayazayazayazazbzayazbzazaybzbyazbzazbzayazay
azazbybzayazazazazayazayazayazbzazazayazazayazazaz
byazazayazazbyazbyazazazbzazazbzazazbzazbybzbzaz
ayazayazazazayazazazazbzayayazazazazbzazazayazazaz
azazazazbzazbyaybzbzazbyazazazazazazazazazazbzaza
aybzazazayazazazazazbzayayayazayayazayayazazayaza
azbzaybzbzazayazbzazazaybzaybzazazazazazayazazaz
ayazbyayaybzazazazayazazazbybzbzazbyazaybzazazbzaz
azazazazbzazazazazazazazazbzazazbyazbzayazazazaz
azayazbzazbzazbyazayayazazbzazazazazazaybzazazaz
azazbzazbyayazayazayazazazazazayazbyayazazazazaza
byaybzazbzbzazazazbzazbzbzbzazazazazazazazazazaz
azbyazazayazayazazazazbzazazazazazbzbzbzbyazayby
aybyazazazaybzazazbzazbzayayazayazbzazazayazaybzby
azbzayazazazayayazazazazayazazayazazayazazazazaza
ayazazazayaybzbzazayayazbyazazazazazazbzazbzbzaz
azazazbzbzazbzbyazbyazayazbzazbyazayayazbzazayaza
ayazazazazayazazbyazbzazazazayazbzazbzazazazazaza
azazazazazazayaybybzazbzayazazayaybzbzayayazayazay
azazazazbzazayazbyazbyazbyazazazazazazazayazayaza
azazazazazazazayazayazayazazazbzazazbyayazazazazbz
azbyazaybybzazazazazazbzazazbyazayazbzazazazbyazay
azazbzbzazayazazazazazazazayazazazazayazazbzazbza
byazazazbzazazazbyazazayazazbzaybzazazazazazayaza
bzazbzbzazayayazbzbzazazbyayaybzazazazbyazazaybza
azayazayazazazazbzazbzayayazazayazazayazazazaybz
azbzazayazazazbzazayazbzaybybzayazazbyazbzayazaza
ayazazayazazazazazazbzayazazazayazazbzazazazazaza
bzbyayazazazazazazayazaybzaybyazazbzayazazbzazby
azbybzazayazazbzazayazazbzazazazbzazazazayazazaz
bzazazaybzazazazazazazbzbzazazbzazazayazbyazayazby
azazazbzazazayayazayaybzaybzazayazbyazazazazazazb
azbzazayazayazayazbzazazbzayazazbzbzazazbzazazazay
azazazazazazazazazazazazazbybyazaybzbzazazazbzaz
azaybzazazaybzazazaybyazazazayayazbzazayaybzazaz
bzazazayazbzazbzbzazayazbzayazazbyazazbzazazazaz
azazazazbzazbzaybzazbzaybzayayayazazazbzayazazbza
ayayayazayaybzbzazayazazbzayazbzbzazbzazazazazaz
azazazazbzazazayazazazazazbzazbyazbzazazazaybyazaz
azazbzazazazazazbzazazazazaybzbzbzazazazbyayazazaz
azbzayazayayazazbzazazazazazazayayazazazbzayazbzaz
bzayazazazayazazazbzayazazazayazazayaybzayazbzbza
azazazazazayazazazazbyazazazbyazbzazazazbzazazaza
azazazayazazbzayazazayazbzbzazaybzazayazazbyayaya
azazazbzazazazbzazbzbyazazazazazazazazayazazazaya
azbzbyazbyayazazazbzayazazaybzazayazazazazayazaza
azayazazbzaybyazbzazayayazbzazayayaybzazazbzazaz
azazbzayazbzazazazbyazazazbzazbzbzayazbyazbyazbzaz
bzazazbzazazazayazazbzazazayazbzazazbzazbyazazay
bzbybyazaybzazbzbzazazazazayazazbzaybzazazazazazb
bzazazazayazazayazazazazazazazazazazazazbzbzazazaz
bzazaybzbzayazaybyazazazazazayazazaybzayazazbzbz
azazbzbzazbzazazazazazazbzazazbzazazazbzazbyazayay
azazbzazazayayayayazazazazazazazazazayayazazazazaz
azazbyayazbzazazbzazbzbzazazazazayazbzazazazayazb
azazayaybzbzbzayazazazazazazayazazbyazayazayazayaz
azazazazbzazazbzazazazbzbzaybzayazazbzbzazaybzaz
